src/testRunner/unittests/helpers/node10Result.ts

```diff
@@ -72,6 +72,8 @@ export function getFsContentsForNode10Result(): FsContents {
         `,
         "/home/src/projects/project/tsconfig.json": JSON.stringify({
             compilerOptions: {
+                target: "es5",

```

Comment:
Instead of setting target i think if you add `[expectedLibFile]: libFile.content` to this map this test would not break so much.